src/main/java/hill/weatherconfidence/ingest/model/GeocodeItem.java

```diff
@@ -0,0 +1,74 @@
+package hill.weatherconfidence.ingest.model;
+
+
+/*
+ * [{"name":"Hemel Hempstead","local_names":{"en":"Hemel Hempstead","ru":"Хемел-Хемпстед"},"lat":51.7511784,"lon":-0.472528,"country":"GB","state":"England"}]
+ * */
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class GeocodeItem {
+
+    private String name;
+    private Double lat;
+    private Double lon;
+    private String country;
+
+    public static GeocodeItem defaultLocation() {
+        // London
+        GeocodeItem item = new GeocodeItem();
+        item.setCountry("UK");
+        item.setLat(Double.parseDouble("51.5073219"));
+        item.setLon(Double.parseDouble("-0.1276474"));
+        item.setCountry("GB");
+        return item;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public Double getLat() {
+        return lat;
+    }
+
+    public void setLat(Double lat) {
+        this.lat = lat;
+    }
+
+    public Double getLon() {
+        return lon;
+    }
+
+    public void setLon(Double lon) {
+        this.lon = lon;
+    }
+
+    public String getCountry() {
+        return country;
+    }
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Need equals() and hashcode() for .equals() to be true for objects that have the same instance property values.  We need to declare that two objects that have the same properties are in fact 'equal'.